src/main/scala/com/training/scala/getstarted/types/ContextBounds.scala

```diff
@@ -0,0 +1,51 @@
+package com.training.scala.getstarted.types
+
+/**
+  * Context Bound is used to specify a implicit value is should be available.
+  *
+  * Eg:
+  *   `def fun[A: B](a: A) = a.get()`
```

Comment:
`def fun[A : B](a: A) = a.get()`, stylistic one. Have seen a space before and after ":"